.github/workflows/build.yml

```diff
@@ -206,7 +206,7 @@ jobs:
 
       - id: deploy
         name: Deploy
-        uses: aneoconsulting/ArmoniK.Action.Deploy/deploy@main
+        uses: aneoconsulting/ArmoniK.Action.Deploy/deploy@dd/fixUploadArtifact
```

Comment:
Revert to main, https://github.com/aneoconsulting/Armonik.Action.Deploy/pull/7 has been merged